File: webapi/StoreManagement/Models/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace StoreManagement.Models
{
    public class Customer
    {
        public int Customer_ID { get; set; }
        public string Customer_Name { get; set; }
        public string Address { get; set; }
        public string Telephone { get; set; }
        public ICollection<Order> Orders { get; set; }
    }
}

File: client/src/app/Models/Customer.ts
import { Role } from "./Role";

export class Customer{
    customer_ID:number;
    customer_Name:string;
    address:string;
    telephone:string;
    role!: Role;
    token?: string;
    constructor(customer_ID:number,customer_Name:string,address:string,telephone:string)
    {
        this.customer_ID = customer_ID,
        this.customer_Name=customer_Name,
        this.address=address,
        this.telephone=telephone;
    }
}